okonomiyaki/platforms/__init__.py

```diff
@@ -1,9 +1,11 @@
 # flake8: noqa
 from .abi import PlatformABI, default_abi
 from .epd_platform import X86, X86_64, EPDPlatform, applies
-from .platform import Platform, OSKind, FamilyKind, NameKind
+from ._platform import Platform, OSKind, FamilyKind, NameKind
```

Comment:
I think that the fact that the `platforms` module was public and now it is not is a change that can break code that depends on previous versions. It also does not look that this change is necessary.